src/main.rs

```diff
@@ -1,8 +1,60 @@
 use geo::Point;
 use serde_json::to_string_pretty;
 use zonebuilder::{clockboard, Params};
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// A basic example
+#[derive(StructOpt, Debug)]
+#[structopt(name = "zb")]
+struct Opt {
+    // A flag, true if used in the command line. Note doc comment will
+    // be used for the help message of the flag. The name of the
+    // argument will be, by default, based on the name of the field.
+    /// Activate debug mode
+    #[structopt(short, long)]
```

Comment:
When I `cargo run`, I get a crash:
```
thread 'main' panicked at 'Argument short must be unique

        -n is already in use', /home/dabreegster/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3/src/app/parser.rs:193:13
```

https://docs.rs/structopt/0.3.21/structopt/#specifying-argument-types
You have `short` on all of these flags, but the docs says it uses the first letter. There's a collision with `num_vertices`, `num_segments` both mapping to `-n`, and some others. You can do `short = "x"` to explicitly name them, or just omit the short form for some of these. If I make that fix locally, then `cargo run -- --help` works as expected

Reply:
>  You have `short` on all of these flags, but the docs says it uses the first letter. There's a collision with `num_vertices`, `num_segments` both mapping to `-n`, and some others. You can do `short = "x"` to explicitly name them, or just omit the short form for some of these. If I make that fix locally, then `cargo run -- --help` works as expected

:facepalm: sorry that was a bit of a silly one, should have checked.